packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -0,0 +1,154 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import { IcButton, IcCard, IcStatusTag, IcTypography } from "../../components";
+import { HAVE_CLASS, HAVE_LENGTH } from "../utils/constants";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+import { should } from "chai";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
+
+describe("IcCard", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("renders", () => {
+    mount(
+      <a href="/">
+        <IcCard heading="Card" message="This is a static card"></IcCard>
+      </a>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("a").should(HAVE_CLASS, "ic-card-wrapper-link");
+    cy.get("a").focus().should("have.focus");
+    cy.compareSnapshot("randerCard", DEFAULT_TEST_THRESHOLD + 0.03);
+    cy.checkA11y(undefined, CYPRESS_AXE_OPTIONS);
```

Comment:
With rebase you will have access to cy.checkA11yWithWait() that should be used instead of all cy.checkA11y() instances.